docs/configuring-crash-stats.rst

```diff
@@ -0,0 +1,310 @@
+.. index:: configuring-crashstats
+
+.. _configuring-crashstats-chapter:
+
+Configuring Crash-Stats and PostgreSQL
+======================================
+
```

Comment:
Right now it's unclear what parts in this section are useful (if any) outside of our use-case. Let's be sure to be absolutely clear about the elements of general vs. Moz interest.